glide-core/src/protobuf/redis_request.proto

```diff
@@ -38,118 +38,553 @@ message Routes {
     /// An unknown command, where all arguments are defined by the user.
-    CustomCommand = 1;
+    CustomCommand                  = 1;
+
+    //// Bitmap commands: https://redis.io/docs/latest/commands/?group=bitmap
+
+    BitCount                       = 101;
+    BitField                       = 102;
+    BitFieldRo                     = 103;
+    BitOp                          = 104;
+    BitPos                         = 105;
+    GetBit                         = 106;
+    SetBit                         = 107;
+
+    //// Cluster commands : https://redis.io/commands/?group=cluster
+
+    Asking                         = 201;
+    ClusterAddSlots                = 202;
+    ClusterAddSlotsRange           = 203;
+    ClusterBumpEpoch               = 204;
+    ClusterCountFailureReports     = 205;
+    ClusterCountKeysInSlot         = 206;
+    ClusterDelSlots                = 207;
+    ClusterDelSlotsRange           = 208;
+    ClusterFailover                = 209;
+    ClusterFlushSlots              = 210;
+    ClusterForget                  = 211;
+    ClusterGetKeysInSlot           = 212;
+    ClusterInfo                    = 213;
+    ClusterKeySlot                 = 214;
+    ClusterLinks                   = 215;
+    ClusterMeet                    = 216;
+    ClusterMyId                    = 217;
+    ClusterMyShardId               = 218;
+    ClusterNodes                   = 219;
+    ClusterReplicas                = 220;
+    ClusterReplicate               = 221;
+    ClusterReset                   = 222;
+    ClusterSaveConfig              = 223;
+    ClusterSetConfigEpoch          = 224;
+    ClusterSetslot                 = 225;
+    ClusterShards                  = 226;
+    ClusterSlaves                  = 227;
+    ClusterSlots                   = 228;
+    ReadOnly                       = 229;
+    ReadWrite                      = 230;
+
+    //// Connection Management commands : https://redis.io/commands/?group=connection
+
+    Auth                           = 301;
+    ClientCaching                  = 302;
+    ClientGetName                  = 303;
+    ClientGetRedir                 = 304;
+    ClientId                       = 305;
+    ClientInfo                     = 306;
+    ClientKillSimple               = 307;
+    ClientKill                     = 308;
+    ClientList                     = 309;
+    ClientNoEvict                  = 310;
+    ClientNoTouch                  = 311;
+    ClientPause                    = 312;
+    ClientReply                    = 313;
+    ClientSetInfo                  = 314;
+    ClientSetName                  = 315;
+    ClientTracking                 = 316;
+    ClientTrackingInfo             = 317;
+    ClientUnblock                  = 318;
+    ClientUnpause                  = 319;
+    Echo                           = 320;
+    Hello                          = 321;
+    Ping                           = 322;
+    Quit                           = 323; // deprecated in 7.2.0
```

Comment:
Do we really need to add the deprecated variants? I guess it doesn't hurt, but I wonder what the AWS side will want.